bin/ssh_scan_worker

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+$:.unshift File.join(File.dirname(__FILE__), "../lib")
+
+require 'ssh_scan'
+
+# An example of a dummy worker process performing scans
+logger = Logger.new(STDOUT)
+
+loop do
+  sleep 0.5
+  worker = SSHScan::Worker.new({:server => "192.168.10.37", :port => 8000})
```

Comment:
Does it make sense to throw the "server" into a config file instead of hardcoded here? I'm assuming this was a leftover artifact from testing?